### PR TITLE
chore: Updates for tree-sitter 0.24

### PR DESCRIPTION
**Pull request by ZedThree**
Latest `tree-sitter-cli` moves some tree-sitter specific metadata out of `package.json` and adds `CMakeLists.txt`

### CODE REVIEW

File: tree-sitter.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        "queries/highlights.scm"
+      ],
+      "injection-regex": "fortran"
+    }
+  ],
+  "metadata": {
+    "version": "0.2.0",
+    "license": "MIT",
+    "description": "Fortran grammar for tree-sitter",
+    "authors": [
```
Comment (stadelmanma) [inline tree-sitter.json:24]: If you feel inclined to you can add yourself to the authors listing, you have really advanced the state of this library.

Comment (ZedThree) [reply tree-sitter.json:24]: Thanks! I've done so, looks like the commit is taking some time to turn up though

**Comment by ZedThree** [comment]
@stadelmanma ok to merge this and make a new release when you're able?

**Comment by stadelmanma** [comment]
Merged and pushed a v0.3.0 tag, I'll try to work on getting the npm release pushed. I need to recover my NPM account.

**Pull request merged**